src/components/stack/_index.scss

```
@@ -0,0 +1,39 @@
+@use '../../scss';
+
+.c-stack {
+	@include scss.component-properties('stack');
+	display: flex;
+}
+
+.c-stack[data-style-direction='vertical'] {
+	flex-flow: column wrap;
```

No we have `flex-wrap` being applied the `flex-flow` does not need the wrap part

```suggestion
	flex-flow: column;
```